lib/integrations/awsoidc/create_ec2ice.go

```diff
@@ -52,6 +49,17 @@ type CreateEC2ICERequest struct {
 	ResourceCreationTags AWSTags
 }
 
+// CreateEC2ICERequestEndpoint contains the information for a single Endpoint to be created.
+
+type CreateEC2ICERequestEndpoint struct {
```

Comment:
Why not just EC2ICEEndpoint?

```suggestion
type EC2ICEEndpoint struct {
```